examples/helloworld/helloworld_test.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+ 
+#  Copyright 2019 gRPC authors.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+# 
+#      http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# 
+
+set +e
+
+clean () {
+    pkill -s 0
+    wait
```

Comment:
I think I understand why the tests are failing in travis.  This wait will set $? to the exit code of the killed server job, which is then used as the exit code of this script.  I think adding a "exit 0" at the bottom will resolve that.  Everything else LGTM, thank you very much for this PR!

Reply:
Added the exit 0 at the bottom of the file but it doesn't seem to fix it. Is it possible that the script isn't able to kill the process and then is stuck waiting? 

Normally when I run the script locally it will print that the job is terminated to the terminal but it doesn't appear to be showing up on travis.

Reply:
It seems like travis's `pkill` is killing the shell process as well as the children.  This works, though (I tested it on travis):

```sh
  jobs -p | xargs -n 1 pkill -P
```

Also, the `exit 0` is not necessary at the end (`wait` exits with success unless there is an error in its own execution), if you prefer to remove it.

Reply:
That seems to work! Thanks for the help :)